addons/web/static/src/js/public/public_crash_manager.js

```diff
@@ -0,0 +1,65 @@
+odoo.define('web.public.CrashManager', function (require) {
+"use strict";
+
+var core = require('web.core');
+var CrashManager = require('web.CrashManager').CrashManager;
+var mixins = require('web.mixins');
+var ServicesMixin = require('web.ServicesMixin');
+var ServiceProviderMixin = require('web.ServiceProviderMixin');
+
+var _t = core._t;
+
+var PublicCrashManager = CrashManager.extend(mixins.EventDispatcherMixin, ServicesMixin, ServiceProviderMixin, {
+    init: function () {
+        mixins.EventDispatcherMixin.init.call(this);
+        this._super.apply(this, arguments);
+        ServiceProviderMixin.init.call(this);
+    },
+
+    notifyConnectionState: function () {
+        var options;
+        if (this.isConnected) {
+            options = {
+                type: 'info',
+                title: _t("Connection restored"),
+                message: _t("You are back online"),
+            };
+        } else {
+            options = {
+                type: 'warning',
+                title: _t("Connection lost"),
+                message: _t("Trying to reconnect..."),
+            };
+        }
+        options.sticky = true;
+        this.displayNotification(options);
+    },
+    show_warning: function (error) {
```

Comment:
Yes this is a possibility but I don't think I am suited to make that decision haha

Reply:
Did you find a case in the backend where this occurs ?

Reply:
Go to the /web page stop odoo-bin and click on an icon, there will be a rpc and no respond so a toast with a diconnected message will appear, then if odoo-bin is started another toast with reconnection message will appear.

Reply:
We are talking about the `show_warning` method here, not the connection/disconnection error ... :)

Reply:
My bad... Yes there is some you can look at the enterprise PR https://github.com/odoo/enterprise/pull/3999 for an example

Reply:
Now that I look at the enterprise part of my changes I should update it to use ```WarningDialog``` instead of ```show_warning``` like everywhere else.

Reply:
@JKE-be Should the toast Notification be the way we display warnings about failed rpc in the backend instead of the Dialog ??

Reply:
In the backend *